src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Data
```

Comment:
better avoid using Data-annotation with an entity, read more about it https://www.jpa-buddy.com/blog/lombok-and-jpa-what-may-go-wrong/
but in general, Lombok is a nice tool.